java/KafkaConnectors/README.md

```diff
@@ -9,35 +9,37 @@ This example demonstrate how to use
 [Flink Kafka Connector](https://nightlies.apache.org/flink/flink-docs-release-1.18/docs/connectors/datastream/kafka/),
 source and sink.
 
-This example uses `KafkaSource` and `KafkaSink`.
+This example uses KafkaSource and KafkaSink.
 
 ![Flink Example](images/flink-example.png),
 
+> In this example, the Kafka Sink uses *exactly-once* delivery guarantees. This leverages Kafka transaction under the hood, improving guarantees but 
+> adding some overhead and increasing the effective latency of the output to the consumers of the destination Kafka topic.
+>
+> We recommend not to consider Kafka Sink *exactly-once* guarantees as a default setting for all sinks to Kafka. 
+> Make sure you understand the implications before enabling it. Refer to the [Flink Kafka sink documentation](https://nightlies.apache.org/flink/flink-docs-release-1.19/docs/connectors/datastream/kafka/#fault-tolerance) for details.
```

Comment:
there is also a known "flaw" in EO Sink still pending FLIP resolution: https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=255071710

Reply:
Added ref to FLIP-319 to the disclaimer.